packages/story-editor/src/components/colorPicker/basicColorList.js

```diff
@@ -75,6 +73,7 @@ function getPatternAsString(pattern) {
 }
 
 function BasicColorList({
+  'aria-labelledby': ariaLabelledBy,
```

Comment:
Hm, doesn't this work as part of the `rest`?

Reply:
Seems to work as-is as well:

<img width="996" alt="Screenshot 2021-11-10 at 11 58 02" src="https://user-images.githubusercontent.com/3294597/141092177-b7640750-e89b-4ecf-9922-3addc61dbe9d.png">
<img width="333" alt="Screenshot 2021-11-10 at 11 59 21" src="https://user-images.githubusercontent.com/3294597/141092090-0208a7fc-37f1-45f8-983d-54132146360a.png">

Thoughts?

Reply:
It used to but the `listbox` was moved off of the top most wrapper: https://github.com/google/web-stories-wp/pull/9673/files#diff-eb2e914ab33bdf308ba539de8ae8c6d6fd708795611ecb60bea4efffa1233647R101-R102

But this won't be necessary with the update to use a `Wrapper` like you suggested